setup.cfg

```
@@ -1,2 +1,48 @@
+[metadata]
+name = robotframework-remoterunner
+version = 1.2.0
+author = Chris Brookes
+author_email = chris-brookes93@outlook.com
+description = A library that provides the ability to execute RobotFramework test suites on a remote host.
+keywords=robotframework automation testautomation rpa testing acceptancetesting atdd bdd
+long_description_content_type = text/markdown
+license = MIT
+url = https://github.com/chrisBrookes93/robotframework-remotrunner
+project_urls =
+    Issue Tracker=https://github.com/chrisBrookes93/robotframework-remotrunner/issues
+    Repository=https://github.com/chrisBrookes93/robotframework-remotrunner
+classifiers =
+    Development Status :: 6 - Mature
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Framework :: Robot Framework
+    Topic :: Software Development :: Testing
+
+[options]
+packages = find:
+python_requires = >=2.7
+install_requires =
+    robotframework < 3.2
+    six
+
+[options.extras_require]
+dev =
+    unittest2
+    mock
+    coverage

```

Please update to:
unittest2==1.1.0
mock==3.0.5
coverage==5.3.1

This is the latest version for coverage and unittest2. 
v3.0.5 is the last release of _mock_ to support Python 2 & 3. Since none of the tests are currently using any features from version 4*, I don't think there's a need to use conditional dependencies here in order to use a later version in py3.

```suggestion
dev =
    unittest2==1.1.0
    mock==3.0.5
    coverage==5.3.1
```